content/webapp/utils/requesting.ts

```diff
@@ -18,10 +19,26 @@ const locationIsRequestable = (location: PhysicalLocation): boolean => {
   );
 };
 
-export const itemIsRequestable = (item: PhysicalItem): boolean => {
+export const itemIsRequestable = (
+  item: PhysicalItem,
+  offsiteRequesting = false
+): boolean => {
   // ok because there is only one physical location in reality
   const physicalLocation = getFirstPhysicalLocation(item);
-  return !!physicalLocation && locationIsRequestable(physicalLocation);
+  // we know an item is offsite/deepstore if its 1st available date is more than 3 days in the future
```

Comment:
Could this happen to a local item that is meant to be returned in 3 days? Or what about bank holiday weekends or Christmas that has two bank hols in a row? (I'm not 100% on how it all works, just thinking of possible edge case)

Reply:
Changed to 9 days, that will work for the 3 day closure at the end of August. Might still be tight for Xmas but by then the toggle will be gone 🙏🙏